opal/templates/modal_base.html

```diff
@@ -34,8 +34,7 @@
   {% endif %}
 
   {% block modal_save %}
-  <button one-click-only class="btn btn-primary" ng-click="save('save')"
-          ng-disabled="form.$invalid || saving">
+  <button check-form="form" class="btn btn-primary" ng-click="form.$valid && save('save')">
     <i class="fa fa-save"></i>
```

Comment:
I've rolled the one-click only logic into the check-form directive as we never want to check a form without the one click.